src/data/learnQuizzes/questionBank.ts

```diff
@@ -0,0 +1,754 @@
+// Import data types
+import { QuestionBank } from "../../types"
+
+// Declare hash map of question bank
+const questionBank: QuestionBank = {
+  // TODO: Use hash of English prompt to generate this key
+  // What is Ethereum?
+  a001: {
+    prompt: "The biggest difference between Ethereum and Bitcoin is:",
+    answers: [
+      {
+        id: "a001-a", // TODO: Use hash of English question label to generate this id
```

Comment:
Why does each answer need an ID - can we not rely on their index?

Reply:
We could potentially, but went with this approach to help prevent any issues is the ordering ever changes (ie, if we decide to shuffle answer choices in the future).